maven-model-builder/src/site/apt/index.apt

```diff
@@ -192,6 +192,10 @@ Maven Model Builder
 *----+------+------+
 | <<<maven.home>>> | The path to the current Maven home. | <<<$\{maven.home\}>>> |
 *----+------+------+
+| <<<maven.rootdir>>> | the directory containing the root <<<pom.xml>>> file of a multi module project, in a single module project this is the same as <<<project.basedir>>> | <<<$\{maven.rootdir\}>>> |
+*----+------+------+
+| <<<maven.topdir>>> | the directory containing the top-level <<<pom.xml>>> in this reactor build | <<<$\{maven.topdir\}>>> |
+*----+------+------+
```

Comment:
Actually, `topdir` should not be part of any model interpolation, because it's mainly the current directory and should definitely not have any effect.  This will require an additional parameter to `ModelInterpolator.interpolate()`  or to add it to the `ModelBuildingRequest`. 

However, `rootdir` should be part of profile activation along `basedir` imho.

Reply:
? `topdir` should be available in interpolation, it is its main usage to reference absolutely configurations otherwise not sure its interest for end users.

Reply:
> ? `topdir` should be available in interpolation, it is its main usage to reference absolutely configurations otherwise not sure its interest for end users.

No, that's would be the `rootdir`.  The `topdir` would be defined as the the top-most directory of executed projects in the reactor (basically, the current directory or the one pointed to when using `-f`/`--file`).

Reply:
what's the point of topdir then? it does not have to match cwd nor the rootdir so sounds like a randomdir at usage

Reply:
`topdir` is not really new, it's mostly a rename of the `executionRootDir` property to make it more coherent with `basedir` and `rootdir`, else we end up with `root directory` and `execution root directory` for different things, and that will definitely lead to problems imho.